content/en/docs/concepts/security/application-security-checklist.md

```diff
@@ -0,0 +1,69 @@
+{{< /caution >}}
+
+<!-- body -->
+
+## Application design
+- [ ] Following the right [design principles for the application.](https://kubernetes.io/blog/2018/03/principles-of-container-app-design/)
+- [ ] Application configured with appropriate {{< glossary_tooltip text="QoS class" term_id="QoS-class" >}} through resource request and limits
+  - [ ] Memory limit is set for the workloads with a limit equal or inferior to the request.
+  - [ ] CPU limit might be set on sensitive workloads.
+
+## Pod level configurations
+Configure {{< glossary_tooltip text="Security Context" term_id="Security-Context" >}} for the pod-container.
+- [ ] `automountServiceAccountToken` should be set to `false` unless the pod specifically requires access to the Kubernetes API to operate.
+
+### Pod SecurityContext
+- [ ] Set `runAsNonRoot: true`.
+- [ ] Configure User permissions and access controls using `runAsUser` and `runAsGroup`.
+- [ ] Optionally add a supplimentary group with `fsGroup` to access persistent volumes.
+- [ ] Appropriate Pod Security Standard policies are enforced to all pods in the application namespace.
+
+### Container Security Context
+- [ ] Disable privilege escalations using `allowPrivilegeEscalation: false`.
+- [ ] Configure the root filesystem to be read-only with `readOnlyRootFilesystem: true`.
+- [ ] Avoid running privileged containers with `privileged: false`.
+- [ ] [Set appropriate Seccomp profiles.](/docs/tasks/configure-pod-container/security-context/#set-the-seccomp-profile-for-a-container)
+- [ ] [Configure appropriate AppArmor policies.](/docs/tutorials/security/apparmor/)
+- [ ] [Assign appropriate SELinux labels to the contianers.](/docs/tasks/configure-pod-container/security-context/#assign-selinux-labels-to-a-container)
+- [ ] Drop all capabilities from the containers, and add back only specific ones that are needed for operation of the container.
+
+## Role Based Access Control (RBAC)
+- [ ] Permissions such as `create`, `patch`, `update` and `delete` should be only granted if necessary.
+- [ ] Avoid creating RBAC permissions to create or update roles which can lead to [privilege escalation](/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping).
+
+The `create`, `update` and `delete` are generally agreed upon to be used judicously. The `patch` permission can [allow users to update labels on the namespace or deployments](/docs/concepts/security/rbac-good-practices/#namespace-modification) making them more vulnarable.
```

Comment:
Another typo here, not a fan either of "making them more vulnerable", maybe something like "allowing them to escalate privilege" or something less stronger but more conveyor of the "attack" sense? Not sure.
```suggestion
The `create`, `update` and `delete` are generally agreed upon to be used judicously. The `patch` permission can [allow users to update labels on the namespace or deployments](/docs/concepts/security/rbac-good-practices/#namespace-modification) making them more vulnerable.
```